example/api_example.py

```diff
@@ -0,0 +1,35 @@
+import base64
+import io
+import requests
+
+address = 'http://127.0.0.1:7860'
+image_file = "path/to/local/image/file"
+im = Image.open(image_file)
+
+img_bytes = io.BytesIO()
+im.save(img_bytes, format='PNG') 
+img_base64 = base64.b64encode(img_bytes.getvalue()).decode('utf-8')
+
+# the most important is this one
+args=[img_base64,True,'0','F:\stable-diffusion-webui\models/roop\inswapper_128.onnx','CodeFormer',1,None,1,'None',False,True]
```

Comment:
Might be useful to add a comment to each of these arguments to explain what it actually does.

Reply:
yes